[BUG] PARSEDATE in Formula not implemented

Using Apache Hop 
I try to parse a Date using the Formula Transform in a Pipeline.
My Formula looks like this:
`PARSEDATE([date],"yyyyMMddHHmmss")`
On running the Pipeline I get the following error message:
`2022/12/17 10:35:10 - Formula.0 - ERROR: Unexpected error
2022/12/17 10:35:10 - Formula.0 - ERROR: org.apache.poi.ss.formula.eval.NotImplementedException: Error evaluating cell Sheet0!A1
2022/12/17 10:35:10 - Formula.0 - 	at org.apache.poi.ss.formula.WorkbookEvaluator.addExceptionInfo(WorkbookEvaluator.java:344)
2022/12/17 10:35:10 - Formula.0 - 	at org.apache.poi.ss.formula.WorkbookEvaluator.evaluateAny(WorkbookEvaluator.java:285)
2022/12/17 10:35:10 - Formula.0 - 	at org.apache.poi.ss.formula.WorkbookEvaluator.evaluate(WorkbookEvaluator.java:216)
2022/12/17 10:35:10 - Formula.0 - 	at org.apache.poi.xssf.usermodel.BaseXSSFFormulaEvaluator.evaluateFormulaCellValue(BaseXSSFFormulaEvaluator.java:56)
2022/12/17 10:35:10 - Formula.0 - 	at org.apache.poi.ss.formula.BaseFormulaEvaluator.evaluate(BaseFormulaEvaluator.java:110)
2022/12/17 10:35:10 - Formula.0 - 	at org.apache.hop.pipeline.transforms.formula.util.FormulaParser.getFormulaValue(FormulaParser.java:98)
2022/12/17 10:35:10 - Formula.0 - 	at org.apache.hop.pipeline.transforms.formula.Formula.processRow(Formula.java:123)
2022/12/17 10:35:10 - Formula.0 - 	at org.apache.hop.pipeline.transform.RunThread.run(RunThread.java:51)
2022/12/17 10:35:10 - Formula.0 - 	at java.base/java.lang.Thread.run(Thread.java:831)
2022/12/17 10:35:10 - Formula.0 - Caused by: org.apache.poi.ss.formula.eval.NotImplementedFunctionException: PARSEDATE
2022/12/17 10:35:10 - Formula.0 - 	at org.apache.poi.ss.formula.UserDefinedFunction.evaluate(UserDefinedFunction.java:56)
2022/12/17 10:35:10 - Formula.0 - 	at org.apache.poi.ss.formula.OperationEvaluatorFactory.evaluate(OperationEvaluatorFactory.java:155)
2022/12/17 10:35:10 - Formula.0 - 	at org.apache.poi.ss.formula.WorkbookEvaluator.evaluateFormula(WorkbookEvaluator.java:541)
2022/12/17 10:35:10 - Formula.0 - 	at org.apache.poi.ss.formula.WorkbookEvaluator.evaluateAny(WorkbookEvaluator.java:275)
2022/12/17 10:35:10 - Formula.0 - 	... 7 more`
which to me indicates that the function I want to call is not implemented. 
As the string defining the format to be parsed is not in the examples I assumed that maybe this special format is not Implemented and changed my formula to the example 
`PARSEDATE("2004/12/25 12:56:45.878";"yyyy/MM/dd HH:mm:ss.SSS")`
taken from the official [website](https://hop.apache.org/manual/latest/pipeline/transforms/formula.html#_datetime) . 
This example has an easy to fix syntax error. The semicolon needs to be replaced with a comma. Nontheless, the errormessage still stays the same, indicating to me, that the PARSEDATE function is not implemented.
Maybe I am missing something here, in that case I would be happy about being pointed towards that issue on my side.

--- a/docs/hop-user-manual/modules/ROOT/pages/pipeline/transforms/formula.adoc
+++ b/docs/hop-user-manual/modules/ROOT/pages/pipeline/transforms/formula.adoc
@@ -537,22 +537,6 @@ Hour=INT(DayFraction*24)
 |NPER([FIELD1]/12, [FIELD2], [FIELD3], [FIELD4], 1) |59.673865|Periods for the investment with the above terms
 |===
 
-==== PARSEDATE
-
-|===
-|*description*|Parses a date using a SimpleDateFormat format string
-|*syntax*|PARSEDATE( Text date; Text format; Text locale; Text timezone )
-|*returns*|Date
-|*semantics*|This computes the date based on the provided format, locale, and timezone.  If timezone is provided, assumes the date is in UTC and converts the date from UTC to the specified timezone.
-|*constraints*|format must be a valid SimpleDateFormat
-|===
-
-*Examples*
-|===
-|expression|result|comment
-|PARSEDATE("2004/12/25 12:56:45.878";"yyyy/MM/dd HH:mm:ss.SSS") |2004-12-25 12:56:45.878|Simple parse.
-|PARSEDATE("2004-12-25"; "yyyy-MM-dd"; "en"; "MDT" ) |2004-12-24 17:00:00.000|2004/12/25 00:00:00 UTC is 2004-12-24 17:00:00 MDT
-|===
 
 ==== PREVWEEKDAY
 

--- a/plugins/transforms/formula/src/main/resources/org/apache/hop/pipeline/transforms/formula/function/functions.xml
+++ b/plugins/transforms/formula/src/main/resources/org/apache/hop/pipeline/transforms/formula/function/functions.xml
@@ -3125,20 +3125,6 @@ System 2    The week containing the first Thursday of the year is the first week
   <!-- Date/Time : this list is complete -->
   <function>
     <category>%Category.DateTime</category>
-    <name>PARSEDATE</name>
-    <description>Parses a date using a SimpleDateFormat format string</description>
-    <syntax>PARSEDATE( Text date, Text format, Text locale, Text timezone )</syntax>
-    <returns>Date</returns>
-    <constraints>format must be a valid SimpleDateFormat</constraints>
-    <semantics>This computes the date based on the provided format, locale, and timezone.  If timezone is provided, assumes the date is in UTC and converts the date from UTC to the specified timezone.</semantics>
-    <examples>
-      <example><expression>PARSEDATE("2004/12/25 12:56:45.878","yyyy/MM/dd HH:mm:ss.SSS")</expression> <result>2004-12-25 12:56:45.878</result> <level>2</level> <comment>Simple parse.</comment></example>
-      <example><expression>PARSEDATE("2004-12-25", "yyyy-MM-dd", "en", "MDT" )</expression> <result>2004-12-24 17:00:00.000</result> <level>2</level> <comment>2004/12/25 00:00:00 UTC is 2004-12-24 17:00:00 MDT</comment></example>
-    </examples>
-  </function>
-
-  <function>
-    <category>%Category.DateTime</category>
     <name>DATETIMEVALUE</name>
     <description>Returns an internal number for a text having a datetime format.</description>
     <syntax>DATETIMEVALUE( Text D )</syntax>

